runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsOverlay.java

```diff
@@ -25,30 +25,14 @@
  */
 package net.runelite.client.plugins.devtools;
 
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Font;
-import java.awt.FontMetrics;
-import java.awt.Graphics2D;
+import java.awt.*;
```

Comment:
Using * for imports is usually discouraged (at least in Java standards) as it reduces explicity of the imports

Reply:
the * is IntelliJs doing, can I disable that somewhere?

Reply:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports#3348855